scipion/install/plugin_funcs.py

```diff
@@ -100,7 +101,7 @@ def hasPipPackage(self):
     def isInstalled(self):
         """Checks if the current plugin is installed (i.e. has pip package).
         NOTE: we might want to change definition of isInstalled, hence the extra function."""
-        reload(pkg_resources)
+        importlib.reload(importlib)
```

Comment:
@pconesa do you know if this will work? I can install plugins successfully.. is there any other way to check?

Reply:
Don't know! Seems a reasonable replacement. I can test it.